### DISCUSSION TITLE
Make many short links for one long link

### DISCUSSION THREAD

**Discussion by Ewgeniy77** [YOURLS/YOURLS#3177] [Q&A] [answered]
Hi! Sorry, maybe I'm inattentive, but after searching a bit, I still couldn't find it. Can you tell me how to YOURLS how to make many different short links for one long link?

**Comment by dgw** [comment, marked as answer]
This is asked and answered fairly often. Try searching more.

(Hint: #3096, #3099)

**Reply by Ewgeniy77** [reply]
Oh, so simple! Thank you! Apparently the fact that English is not my first language I could not formulate the right question. )
Thanks for the answer!
